provider/rancher/rancher.go

```diff
@@ -451,6 +455,12 @@ func (lbp *LBProvider) cleanupLBService(lb *client.LoadBalancerService, lbConfig
 		return nil
 	}
 
+	if stickinessPolicyChanged(lb.LbConfig, lbConfig) {
```

Comment:
There is no need to remove the LB on a stickiness policy change; it is a dynamic parameter that can be updated for existing LB. Could you change it? Most likely it should go here: https://github.com/rancher/lb-controller/blob/c7ce1a075742d19fb13568747bea2b25bd480fb7/provider/rancher/rancher.go#L375